conf/test.config

```diff
@@ -57,4 +57,10 @@ process {
     withName: MEGAN_RMA2INFO_KRONA {
         maxForks = 1
     }
+    withName: 'EIDO_VALIDATE' {
+        ext.args = '--st-index sample'
+    }
+    withName: 'EIDO_CONVERT' {
+        ext.args = '--st-index sample'
+    }
```

Comment:
This we want placed in `modules.conf`, as this will always apply for out pipeline